report/test/renderer/performance-category-renderer-test.js

```diff
@@ -360,4 +360,44 @@ Array [
       });
     });
   });
+
+  describe('_reorderMetrics', () => {
+    it('orders metrics by row', () => {
+      const metrics = [
+        {id: 'first-contentful-paint'},
+        {id: 'speed-index'},
+        {id: 'largest-contentful-paint'},
+        {id: 'interactive'},
+        {id: 'total-blocking-time'},
+        {id: 'cumulative-layout-shift'},
+      ];
+      const orderedMetrics = renderer._reorderMetrics(metrics);
```

Comment:
we don't really care about how the metrics get in the right order (that `_reorderMetrics` was called), just that they're in the right order in the DOM at the end of render. What do you think about adapting the existing `renders the metrics` audit to test this? If we ever do change the ordering implementation ideally the tests would pass without change so long as the desired order also didn't change

Reply:
I think the test for odd # of metrics is still useful at least so I think we should keep these tests still. I'll add an expectation like you mentioned.